codeclimate-wrapper

```diff
@@ -18,32 +18,52 @@ EOF
   exit 1
 }
 
+docker_for_mac() {
+  test -S /var/tmp/docker.sock > /dev/null 2>&1
+}
+
+socket_exists() {
+  test -S $socket_path > /dev/null 2>&1
+}
+
+socket_path() {
```

Comment:
> At least one of them is false in my opinion. My view is that the socket path should be changed.

Makes sense. It also wouldn't work because we'd have to mount in both the symlink and the source for it to work.

Separately, I've gotten the green-light to remove the `boot2docker` code entirely -- which I'll now do.

Are we in agreement then that the few lines I've been pasting is the answer? Just adjust `$socket_path` up top and leave the rest of the script alone?